docs/4.3/admin-guide.md

```diff
@@ -481,7 +481,8 @@ connector. There are three possible values (types) of 2FA:
      or [Authy](https://www.authy.com/) or any other TOTP client.
 
   + `u2f` implements [U2F](https://en.wikipedia.org/wiki/Universal_2nd_Factor)
-    standard for utilizing hardware (USB) keys for second factor.
+    standard for utilizing hardware (USB) keys for second factor. You can use [YubiKeys](https://www.yubico.com/),
+   [solokeys](https://solokeys.com/) or any other hardware token.
```

Comment:
`solokeys` -> `SoloKeys` I think

`any other hardware token` -> `any other hardware token which implements the FIDO U2F standard`?